edit/edit.go

```diff
@@ -392,19 +393,22 @@ func move(ed *Editor, src, dst Address) (addr, error) {
 // It is an error if such a subexpression has more than MaxRunes.
 // \n is a literal newline.
 // If g is true then all matches in the address range are substituted.
+// The n-th address in the range is substituted.
+// If n is not equal to '1' g must be false.
```

Comment:
It's no longer true, removed the 'if n is not equal to '1' g must be false' comment.